vendor/bat-native-ads/src/bat/ads/internal/transfer/transfer.h

```diff
@@ -47,6 +47,7 @@ class Transfer final : public TabManagerObserver {
  private:
   void TransferAd(int32_t tab_id, const std::vector<GURL>& redirect_chain);
   void OnTransferAd(int32_t tab_id, const std::vector<GURL>& redirect_chain);
+  void OnLogAdEvent(const AdInfo& ad, const bool success);
```

Comment:
nit: we can remove const in declaration, thanks

Reply:
fixed